dist/profile/templates/kubernetes/resources/evergreen/service.yaml.erb

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: evergreen
+  labels:
+    app: evergreen
+    type: evergreen
+spec:
+  sessionAffinity: ClientIP
```

Comment:
do you need a session affinity?
The accountapp is the only application that need basically it ensure that a client wlll always be routed to the same pod